frlg-any-percent-beginner/route.json

```diff
@@ -0,0 +1,13 @@
+{
+    "title": "frlg-any-percent-beginner",
```

Comment:
sorry, the title here should still be `Any% (Beginner)`, I just meant to change the folder name. didn't mean to be confusing 😓

Reply:
keep the folder name to what you changed it to that's perfect.